base/system/diskpart/diskpart_msg.mc

```diff
@@ -2816,28 +2827,29 @@ Example:
     DELETE PARTITION
 .
 Language=Romanian
-Deletes the partition with focus.
-
-Syntax:  DELETE PARTITION [NOERR] [OVERRIDE]
+Șterge partiția în cauză.
 
-    NOERR       For scripting only. When an error is encountered, DiskPart
-                continues to process commands as if the error did not occur.
-                Without the NOERR parameter, an error causes DiskPart to exit
-                with an error code.
+Sintaxă:  DELETE PARTITION [NOERR] [OVERRIDE]
 
-    OVERRIDE    Enables DiskPart to delete any partition regardless of type.
-                Typically, DiskPart only permits you to delete known data
-                partitions.
+    NOERR       Doar pentru automatizare. Când apare o eroare, DiskPart
```

Comment:
"For scripting only" gets translated here as "doar pentru automatizare", but in other instances as "doar pentru scriere de fișiere script". Please take care of consistency.